sdk/javascript/test/utils/transforms_spec.js

```diff
@@ -0,0 +1,16 @@
+const { hexToUint8 } = require('../../src/utils/converter');
+const { ripemdKeccak256 } = require('../../src/utils/transforms');
+const { expect } = require('chai');
+
+describe('transforms', () => {
+	it('can transform with ripemd keccak 256', () => {
+		// Arrange:
+		const payload = hexToUint8('BE0B4CF546B7B4F4BBFCFF9F574FDA527C07A53D3FC76F8BB7DB746F8E8E0A9F');
```

Comment:
would rather use one of existing pub keys:
`75D8BB873DA8F5CCA741435DE76A46AFC2840803EBF080E931195B048D77F88C` - from cpp tests
`D6C3845431236C5A5A907A9E45BD60DA0E12EFD350B970E7F58E3499E2E7A2F0` - from js and py tests

Reply:
used key taken from examples

Reply:
yeah, but that key is not used to derive address :D
the two above are

Reply:
idk, that key is actually hashed for sorting, so i thought it was better pick